styles.css

```diff
@@ -0,0 +1,705 @@
+  align-items: center;
+  gap: 0.6rem;
+}
+
+.main--about--info {
+  padding: 20px;
+}
+
+.main--about--header--text {
+  color: #838383;
+  text-align: center;
+  font-size: 1rem;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 1.3125rem;
+  padding: 15px 0;
+}
+
+.main--program--header,
+.main--about--header {
+  color: #fff;
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  position: relative;
+}
+
+.main--about--header {
+  color: #2c2c2c;
+  text-align: center;
+  font-size: 1.25rem;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 1.75rem;
+}
+
+.featured--speaker--header {
+  color: #000;
+  font-size: 1.25rem;
+}
+
+.main--program--header--line,
+.featured--speaker--header--line,
+.main--about--header--line,
+.partners--header--line {
+  width: 1.875rem;
+  height: 0.125rem;
+  height: 3px;
+  background: #a04038;
+  border-radius: 10px;
+}
+
+.main--program--cards--container {
+  margin-top: 20px;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.main--program--card {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 2rem;
+  background-color: rgba(59, 62, 69, 1);
+  padding: 10px;
+}
+
+.cards--icon--text p {
+  color: #ff5438;
+  font-size: 0.89194rem;
+  font-weight: 700;
+  line-height: 1.56094rem;
+}
+
+.main--program--card-description {
+  color: #fff;
+  font-size: 0.66894rem;
+  font-weight: 700;
+  line-height: 1.39369rem;
+}
+
+.cards--icon--text {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 0.55744rem;
+}
+
+.main--program--link {
+  background: #ff4f38;
+  width: 80%;
+  padding: 20px;
+  display: block;
+  margin: 30px  auto 0 auto;
+  text-align: center;
+  color: #fff;
+}
+
+.about--summit--info {
+  padding: 1.25rem 0.75rem;
+  margin: 2.19rem 0;
+  background-color: #fff;
+}
+
+.about--summit--info p {
+  color: #000;
+  text-align: center;
+  font-size: 0.875rem;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 1.5625rem;
+}
+
+.about-contact--info--mail {
+  color: #5f5f5f;
+  text-align: center;
+  font-size: 0.75rem;
+  font-style: normal;
+  font-weight: 800;
+  line-height: 1.5625rem;
+  letter-spacing: 0.015rem;
+  text-decoration-line: underline;
+}
+
+.about--contact--section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(139, 139, 139, 0.072);
+  padding: 1.25rem 0.75rem;
+}
+
+.about--contact--info {
+  color: #7a7a7a;
+  text-align: center;
+  font-size: 0.75rem;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 1.5625rem;
+}
+
+.featured--speaker {
+  padding: 20px;
+}
+
+.featured--speakers--container {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  margin-top: 30px;
+}
+
+.featured--speakers-card {
+  display: flex;
+  gap: 2rem;
+}
+
+.speakers--img img {
+  width: 150px;
+  height: 170px;
+  object-fit: cover;
+}
+
+.speaker--description h4 {
+  color: #000;
```

Comment:
- [ ] please use the correct font color for all the speaker description text. `#272A31`